### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by najibullahjafari**
Here:
- I created a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:

- [ ] List all books.
- [ ] List all people.
- [ ] Create a person (teacher or student, not a plain Person).
- [ ] Create a book.
- [ ] Create a rental.
- [ ] List all rentals for a given person id.

- In my `main.rb`, I defined the entry point, this will be a method called main that is invoked at the end of your file. This method does the following:
- [ ] It presents the user with a list of options to perform.
- [ ] It Lets users choose an option.
- [ ] If needed, It ask for parameters for the option.
- [ ] It has a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  date = gets.chomp
+  book = books.find { |b| b.title == book_title }
+  person = people.find { |p| p.name == person_name }
+  if book && person
+    rental = create_rental(book, person, date)
+    rentals << rental
+  else
+    puts 'Book or person not found.'
+  end
+end
```
Comment (Whoistolu) [inline main.rb]: - [ ] [REQUIRED] Kindly ensure that a user is able to create rental. Currently, a user is not able to create rental.
Screenshot 2023-09-15 at 15 59 32

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+def create_person_option(people)
+  print 'Enter person name: '
+  name = gets.chomp
+  print 'Enter person type (teacher or student): '
+  type = gets.chomp.downcase
+  print 'Enter ID for person: '
+  id = gets.chomp.to_i
+  person = create_person(id, name, type)
+  people << person if person
+end
```
Comment (Whoistolu) [inline main.rb]: - [ ] [REQUIRED] Kindly ensure that you replicate the design as in the video guideline provided when a user tries to create a person.  You skipped the age of the person, you also forgot to add options 1 and 2 to the student and teacher, and you also forgot to ask for parent permission for the student option. Let's implement these recommendations before asking for another review.
### Your Design
Screenshot 2023-09-15 at 16 07 36

### The guideline provided
Screenshot 2023-09-15 at 16 05 36

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @najibullahjafari 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlight
🔆 `main.rb` the entry point is defined. The method called `main` is invoked at the end of the file

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @najibullahjafari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Your app works as expected ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**